Casks/f/foks.rb

```diff
@@ -17,7 +17,8 @@
     end
   end
 
-  binary "foks"
+  binary "foks", target: "foks"
```

Comment:
```suggestion
  binary "foks"
```

Adding `target:` is not required here, since the defaults are used.